ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/forkchoice/ForkChoiceUpdateData.java

```diff
@@ -196,6 +190,44 @@ public SafeFuture<Optional<ForkChoiceUpdatedResult>> send(
     return forkChoiceUpdatedResult.thenApply(Optional::of);
   }
 
+  private void logSendForkChoiceUpdatedComplete(Optional<Bytes8> payloadId) {
+    if (LOG.isDebugEnabled()) {
+      LOG.debug(
+          "send - forkChoiceUpdated returned payload id {} for {}, {}",
+          payloadId,
+          forkChoiceState,
+          payloadBuildingAttributes);
+    } else {
+      if (payloadBuildingAttributes.isPresent()) {
+        LOG.info(
+            "Builder returned payload id {}, (block slot: {})",
+            payloadId.isPresent() ? payloadId.get() : "EMPTY",
+            payloadBuildingAttributes.get().getBlockSlot());
+      } else {
+        LOG.info(
+            "Local execution layer returned payload id {}",
+            payloadId.isPresent() ? payloadId.get() : "EMPTY");
+      }
+    }
+  }
+
+  private void logSendingForkChoiceUpdated() {
+    if (LOG.isDebugEnabled()) {
+      LOG.debug(
+          "send - calling forkChoiceUpdated({}, {})", forkChoiceState, payloadBuildingAttributes);
+    } else {
+      if (payloadBuildingAttributes.isEmpty()) {
+        LOG.info(
+            "Calling local execution layer to start block production (parent slot: {})",
+            forkChoiceState.getHeadBlockSlot());
```

Comment:
So this is why i said parent slot, because if we're building on the headBlockSlot, then that would be the parent of what we're building, so i guess this is just overall badly worded...